### ISSUE TITLE
Object inspection depth is too shallow in logs

### ISSUE THREAD

**Issue by alexsaveliev** [sourcegraph/javascript-typescript-langserver#117]
seen the following entry when running with `-t` 

```
LOG  <-- { jsonrpc: '2.0',
 id: 3,
 result:
  { contents: [ [Object], 'TypeScript service under test ' ],
    range: { start: [Object], end: [Object] } } }
```

Maybe it's worth to print JSON representation of requests/responses? It may help us to reproduce issue by sending JSON data using curl

**Comment by felixfbecker** [comment]
> Maybe it's worth to print JSON representation of requests/responses? It may help us to reproduce issue by sending JSON data using curl

Isn't curl only for HTTP?

**Comment by alexsaveliev** [comment]
Okay, `curl` may be not the best choice because it can only submit one request but still, I think that JSON is still better if we need to re-enact some situation (the best is probably content-length header + payload forming complete HTTP message)

**Comment by felixfbecker** [comment]
You mean a complete LSP message?

**Comment by alexsaveliev** [comment]
No, I mean HTTP one, with HTTP headers and body. Such messages may be concatenated into single file and passed as an input of language-server-stdio

**Comment by felixfbecker** [comment]
I think you are confusing HTTP with LSP: https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#base-protocol

But still, I don't think it would work out that well because to reproduce you would need the whole request sequence including initialize, so you can't just copy+paste because of the log prefixes and other logs.

**Comment by alexsaveliev** [comment]
> I think you are confusing HTTP with LSP: https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#base-protocol

Easy to confuse, isn't? :smile: 

Anyway, I will accept any log message that will give me full LSP message content

**Comment by felixfbecker** [comment]
Yeah, it's definitely inspired by HTTP, but there are subtle differences in headers.

**Comment by alexsaveliev** [comment]
claimed fixed by #122
